### ISSUE TITLE
Font Size Rendering Bug

### ISSUE THREAD

**Issue by gsoli** [CommandFusion/iViewer-For-Android#101]
I'm running a Nexus 7 with OS v4.2.2.  

All text on the panel renders at the same size regardless of  font size in GuiDesigner.  So, if I have one text element set to 12pts and one set to 20pts, they both render at the same size.  Changing the font size in the system preferences for the tablet will change the size of the text in the GUI after you reload it, but it changes all elements.  Changing the Bold flag in GuiDesigner does reflect properly in the GUI on the tablet.

I've tried a few different fonts, including Arial so it doesn't appear to be font-specific.

**Comment by u7310330** [comment]
I am using S2 and one device from china I don't know the brand both are have same problem.

**Comment by fpillet** [comment]
There are only two fonts bundled with android (Roboto and Droid Sans). Have you tried these?

**Comment by GregSoli** [comment]
I installed the Roboto font and tried a few different styles but the fonts still render at the same size.  Also, I tried using the Device Font Chooser in GuiDesigner as some of the fonts wouldn't work with the Windows Font Chooser - kept saying they weren't TTF fonts but they are - and the Device Font Chooser doesn't appear to work as none of the dropboxes are populated with info.  Haven't asked Jarrod about it yet.
